frontend/package.json

```diff
@@ -2,18 +2,11 @@
   "name": "trase-frontend",
   "version": "1.0.0",
   "description": "",
-  "main": "server.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "npm run webpack:server",
-    "dev": "npm run start",
-    "build": "npm run webpack:prod && cp -a public/. dist/",
-    "webpack:prod": "NODE_ENV='production' webpack --config config/webpack.js -p",
-    "webpack:server": "webpack-dev-server --inline --port 8081 --host 0.0.0.0 --content-base public/ --config config/webpack.js",
```

Comment:
Same here, I would keep `"webpack:server` and `start` separated. This way it's easy at a first glance to identify what `npm start` does.